UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/Layer.cs

```diff
@@ -154,17 +154,18 @@ private void OnRotate(MatrixRotationInfo info)
 		List<TileType> excludeTiles = null)
 	{
 		// There's not tile here so its passable.
-		if (!tilemap.HasTile(to))
-			return true;
+		// if (!tilemap.HasTile(to))
```

Comment:
This all supposed to be commented out?

Reply:
yeah, handled in meta-tile map tho still needed for the objects layer since the object layer is special for some reason, actually I can improve it